src/content/configuration/cli.mdx

```diff
@@ -48,26 +62,49 @@ Optionally, you can install `chromatic` as a dependency while using the same scr
   </TabItem>
 </Tabs>
 
-If you don't install `chromatic` as a dependency, `npx` will automatically download and run the latest version. This has pros and cons:
+## Configuring the CLI
 
-- 👍 You'll never be out of date, you'll use the latest version every time, never have to worry about upgrading Chromatic.
-- 👍 You won't need to install the package during local development if you're only running it in continuous integration.
-- 👎 It will be slower to run because the package has to be downloaded first.
+Chromatic CLI can be configured through options in `./chromatic.config.json` file (recommended) placed at the root of your project folder and/or by passing CLI flags.
 
-## Configuration
+<div class="aside">
+  Note: some options are only available as flags. For a full list of available
+  options, please refer to the [configuration options](/configuration-options)
+  page.
+</div>
```

Comment:
@winkerVSbecks, sorry for not catching this earlier, but it seems this URL is incorrect. Shouldn't it be `/docs/configure` instead?